src/template/wasm2c.declarations.c

```diff
@@ -494,20 +494,20 @@ static inline void funcref_table_init(wasm_rt_funcref_table_t* dest,
   }
 }
 
-// Currently we only support initializing externref tables with ref.null.
+// Currently Wasm only supports initializing externref tables with ref.null.
 static inline void externref_table_init(wasm_rt_externref_table_t* dest,
-                                        const wasm_rt_externref_t* src,
+                                        const bool* unused,
```

Comment:
Why not just drop the second argument for now?

Reply:
Done.